src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/ExceptionHandlingController.java

```diff
@@ -194,7 +194,7 @@
     @ExceptionHandler(NotExistException.class)
     public ResponseEntity<JsonResult> exceptionHandler(NotExistException ex) {
 
-        LOGGER.error(ex.getMessage());
+        LOGGER.error(ex.getMessage(), ex);
```

Comment:
Each time you reach not existing study by just typing ID into navbar, you'll have stacktrace in logs, don't you?